src/engines/html5/media-source/media-source-provider.js

```diff
@@ -95,7 +96,7 @@ export default class MediaSourceProvider {
   static getMediaSourceAdapter(videoElement: HTMLVideoElement, source: Source, config: Object): ?IMediaSourceAdapter {
     if (videoElement && source && config) {
       if (!MediaSourceProvider._selectedAdapter) {
-        MediaSourceProvider.canPlaySource(source);
+        MediaSourceProvider.canPlaySource(source, false);
```

Comment:
why false?

Reply:
why do we even have this check here? we should be getting here after we already have a selectedAdapter anyway.
I'll change to true for consistency, but this seems unnecessary.

Reply:
This is api. You can asssume nothing how getting here. 
Why true/false. You have the param from config